### PR TITLE
Fix the button icon sizes

### PR DESCRIPTION
**Pull request by hellcp**
A small fix to make the icon sizes consistent with the mockups

### CODE REVIEW

**Review by DreadBoy — COMMENTED** [review]
As said on Discord, please double check in Flutter dev tools if rendered icons are actually 21x21. I might had a reason why I chose to reduce the size by 2 pixels on each side or it might be a bug. I don't know 🤷‍♂️

Otherwise code is good.

**Comment by hellcp** [comment]
According to the dev tools, the rendered icons are indeed 21x21, and after this patch they do become 25x25

File: lib/ui/common/components/cobble_button.dart [hunk 1]
```diff
@@ -33,7 +33,7 @@ class CobbleButton extends StatelessWidget {
     Widget child = Row(
       mainAxisSize: MainAxisSize.min,
       children: [
-        if (icon is IconData) Icon(icon, size: 21),
+        if (icon is IconData) Icon(icon, size: 25),
```
Comment (DreadBoy) [inline lib/ui/common/components/cobble_button.dart:36]: Default icon size is already 25, as defined here:
https://github.com/pebble-dev/mobile-app/blob/master/lib/ui/theme/cobble_theme.dart#L149
I tried removing `size: 25` in my local branch and indeed it takes global value. I think it's better for this PR to simply remove both parameters and allow global theme to defined icon size.

**Review by DreadBoy — CHANGES_REQUESTED** [review]

File: lib/ui/common/components/cobble_button.dart (hunk 1 above)
Comment (hellcp) [reply lib/ui/common/components/cobble_button.dart:36]: Yeah, that does seem like a better option, I will fix that right away

**Comment by DreadBoy** [comment]
@matejdro This PR is approved, you can merge it.

**Pull request merged**